### PR TITLE
Saying yes without context

### PR DESCRIPTION
**Pull request by justinorringer**
```
Your input ->  yes
I am not able to assist with that topic.
? Here are a few things I can help you with.
Select an option below or type in your questions: Type out your own message...
Your input ->  closing
I am not able to assist with that topic.
? Here are a few things I can help you with.
Select an option below or type in your questions: Type out your own message...
Your input ->  im done
Okay, we can end this conversation.
How would you rate our conversation?
Custom json:
{
  "type": "command",
  "command": "thumbs"
}
Your input ->  yes
Great!
? Would you like to leave any additional feedback? 2: No (/intent_core_no)
? Is there anything else I can help you with today? 1: Yes (/intent_core_yes)
How can I help you?
Your input ->
```

### CODE REVIEW

File: data/stories.yml [hunk 1]
```diff
@@ -38,6 +38,7 @@ stories:
       - or:
           - intent: nlu_fallback
           - intent: intent_unknown_topic
+          - intent: intent_core_yes
```
Comment (justinorringer) [inline data/stories.yml:41]: Should we be adding this for all intents that are just used as answers???

If I say no, I get a weird response too. Without this addition I got no response :face_exhaling: 

```
How can I help you today?
Your input ->  no
How would you rate our conversation?
Custom json:
{
  "type": "command",
  "command": "thumbs"
}
Your input ->
```

Comment (josejulio) [reply data/stories.yml:41]: lets add no as well - I'm guessing this is a side-effect of using stories. apparently stories do stitch together at train time to build new stories and handle more complicated cases

Comment (justinorringer) [reply data/stories.yml:41]: I almost wish there were priorities or something with these definitions. How do conflicts with story/rules get decided/overriden :shrug:

Anyway, that worked for "no", too.

```
Your input ->  no
I am not able to assist with that topic.
? Here are a few things I can help you with.
Select an option below or type in your questions: Type out your own message...
Your input ->  end conversation
Okay, we can end this conversation.
How would you rate our conversation?
Custom json:
{
  "type": "command",
  "command": "thumbs"
}
Your input ->  no
Thanks for sharing.
? Would you like to submit any feedback? Type out your own message...
Your input ->  no
? Is there anything else I can help you with today? Type out your own message...
Your input ->  no
Great. You can start a new conversation with me any time. Thanks for chatting with me today.
Custom json:
{
  "type": "command",
  "command": "core_finish_conversation"
}
Your input ->
```

**Review by josejulio — APPROVED** [review]
lgtm

**Pull request merged**